components/org.wso2.carbon.identity.data.publisher.oauth/src/main/java/org/wso2/carbon/identity/data/publisher/oauth/listener/SAML2BearerGrantAuditLogger.java

```diff
@@ -76,6 +78,16 @@ public void onPostTokenIssue(OAuth2AccessTokenReqDTO tokenReqDTO,
             authenticatedUserTenantDomain = tokReqMsgCtx.getAuthorizedUser().getTenantDomain();
             auditResult = FrameworkConstants.AUDIT_SUCCESS;
 
+            if (LoggerUtils.isLogMaskingEnable) {
+                String maskedUsername = LoggerUtils.getMaskedContent(requestInitiator);
+                if (StringUtils.isNotBlank(requestInitiator) && StringUtils.isNotBlank(authenticatedUserTenantDomain)) {
+                    requestInitiator = IdentityUtil.getInitiatorId(requestInitiator, authenticatedUserTenantDomain);
+                }
+                if (StringUtils.isBlank(requestInitiator)) {
+                    requestInitiator = maskedUsername;
```

Comment:
Same as above we can assign the `LoggerUtils.getMaskedContent(requestInitiator)` directly here